reversestring_advanced-MBA/prompt.md

```
@@ -0,0 +1,125 @@
+
+* Compatibilidad con navegadores
+
+* Cualquier otro detalle que consideres relevante
+
+Una vez tengas toda la información necesaria, desarrolla una aplicación web que invierta cadenas de texto siguiendo estas fases:
+
+## Fase 1: Funcionalidad básica
+
+1. Crea una interfaz con:
+
+  * **Título atractivo y divertido** relacionado con la inversión de texto
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)